android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -50,1235 +50,773 @@ import com.bitmovin.player.api.ui.UiConfig
 import com.bitmovin.player.reactnative.BitmovinCastManagerOptions
 import com.bitmovin.player.reactnative.RNBufferLevels
 import com.bitmovin.player.reactnative.RNPlayerViewConfigWrapper
+import com.bitmovin.player.reactnative.extensions.get
 import com.bitmovin.player.reactnative.extensions.getBooleanOrNull
+import com.bitmovin.player.reactnative.extensions.getDoubleOrNull
 import com.bitmovin.player.reactnative.extensions.getName
 import com.bitmovin.player.reactnative.extensions.getOrDefault
-import com.bitmovin.player.reactnative.extensions.getProperty
 import com.bitmovin.player.reactnative.extensions.putBoolean
 import com.bitmovin.player.reactnative.extensions.putDouble
 import com.bitmovin.player.reactnative.extensions.putInt
-import com.bitmovin.player.reactnative.extensions.setProperty
+import com.bitmovin.player.reactnative.extensions.set
 import com.bitmovin.player.reactnative.extensions.toList
 import com.bitmovin.player.reactnative.extensions.toReadableArray
 import com.bitmovin.player.reactnative.extensions.toReadableMap
-import com.bitmovin.player.reactnative.ui.RNPictureInPictureHandler
+import com.bitmovin.player.reactnative.extensions.withBoolean
+import com.bitmovin.player.reactnative.extensions.withDouble
+import com.bitmovin.player.reactnative.extensions.withInt
+import com.bitmovin.player.reactnative.extensions.withMap
+import com.bitmovin.player.reactnative.extensions.withString
+import com.bitmovin.player.reactnative.ui.RNPictureInPictureHandler.PictureInPictureConfig
 import com.facebook.react.bridge.*
 import java.util.UUID
 
 /**
- * Helper class to gather all conversion methods between JS -> Native objects.
+ * Converts an arbitrary `json` to `PlayerConfig`.
  */
-class JsonConverter {
-    companion object {
-        /**
-         * Converts an arbitrary `json` to `PlayerConfig`.
-         * @param json JS object representing the `PlayerConfig`.
-         * @return The generated `PlayerConfig` if successful, `null` otherwise.
```

Comment:
The `@param` and `@return` documentation was only describing the type and name of the parameter, which is against our style guide. They have thus been removed.